app/javascript/apps/ArtsyPullRequests/ArtsyPullRequests.spec.tsx

```diff
@@ -27,10 +25,9 @@ describe("rendering", () => {
       pullRequests: [],
     }
 
-    const wrapper = mount(<ArtsyPullRequests {...props} />)
+    render(<ArtsyPullRequests {...props} />)
 
-    expect(wrapper.find(PullList)).toHaveLength(1)
-    expect(wrapper.find(PullItem)).toHaveLength(0)
+    expect(screen.queryByRole("link")).toBeNull()
```

Comment:
Ultimately this component gives back a bunch of links to pull requests. That's really the thing we care about testing. 

`queryBy` is a common operation in `react-testing-library` that means give me the element (only one) or give me null. If there are multiple, it'll be an error. `queryByRole` means give me the element based off of its html accessibly role


Refer to the query types docs: https://testing-library.com/docs/queries/about/#types-of-queries.
Also, given that there's a few different types of queries, they have a cheatsheet that is _really_ helpful: https://testing-library.com/docs/dom-testing-library/cheatsheet/#queries